test/include/graph_test/graph_test.h

```diff
@@ -157,10 +164,24 @@ class DBTest : public BaseGraphTest {
         initGraph();
     }
 
-    inline void runTest(const std::vector<std::unique_ptr<TestStatement>>& statements) {
-        TestRunner::runTest(statements, *conn, databasePath);
+    inline void runTest(const std::vector<std::unique_ptr<TestStatement>>& statements,
```

Comment:
put implementation in cpp.

Personally I'm not a big fun of using default parameters. Usually I just overload the function, i.e.
```
runTest(statements);
runTest(statements, checkpointWaitTimeout, connNames);
```
But you can decide this.